### DISCUSSION TITLE
bug? caprover started using caprover.com cert  instead of letsencrypt issued one...

### DISCUSSION THREAD

**Discussion by andrejskvorc** [caprover/caprover#2204] [General]
so now i just saw this... this was my third try install and i was always wondering why it just stops working then i saw this (after few minutes of working it just changes ssl) ... this is 3rd fresh install... usign instructions from site...

so https and redirect is turned on... accessing on base.mydomain.top:3000

on web
"The connection for this site is not secure
base.mydomain.top sent an invalid response.
ERR_SSL_PROTOCOL_ERROR"

when i go to base.mydomain.top 

Your connection isn't private
Attackers might be trying to steal your information from base.mydomain.top (for example, passwords, messages, or credit cards). [Learn more about this warning](chrome-error://chromewebdata/#)
net::ERR_CERT_AUTHORITY_INVALID
This server couldn't prove that it's base.mydomain.top; its security certificate is not trusted by your computer's operating system. This may be caused by a misconfiguration or an attacker intercepting your connection.

Continue to base.mydomain.top (unsafe)

i click continue and get

Nothing here yet :/

and when i inspect certificate

![image](https://github.com/user-attachments/assets/abb158cb-0263-4032-9d2b-a966c4627ae0)

so as i said.. it was working... and i enabled https, added ired mail as an app enabled https there also.... thats all that was done after install

**Comment by githubsaturn** [comment]
Continuing discussion here: https://github.com/caprover/caprover/issues/2205
